thiago_couto/class2/compare.rb

```diff
@@ -0,0 +1,12 @@
+puts "What number would you like to compare?"
+num = gets.chomp.to_i
+
+def compare(a)
```

Comment:
Nice! I'm always interested to see how people use the modulo operator.

My answer to this is always:

```
def compare(a)
    if a % 2 == 0
        puts "your number is even!"
    else
        puts "your number is odd!"
    end
end
```

About the same number of lines, no better or worse!